src/main/java/frc/robot/commands/gpm/ShootKnownPos.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.commands.gpm;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.ArmToPos;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.gpm.Arm;
+import frc.robot.subsystems.gpm.Shooter;
+import frc.robot.subsystems.gpm.StorageIndex; 
+
+public class ShootKnownPos extends SequentialCommandGroup {
+	
+	/**
+	* A preset shot that can be taken by the {@link frc.robot.commands.gpm.ShootKnownPos} command
+	*/
+	public enum ShotPosition {
+		// TODO: add actual values
+		SUBWOOFER_TOP(0.0, 0.0),
+		SUBWOOFER_MIDDLE(0.0, 0.0),
+		SUBWOOFER_BOTTOM(0.0, 0.0);
+		
+		private double armAngle;
+		private double shooterSpeed;
+
+		ShotPosition(double armAngle, double shooterSpeed) {
+			this.armAngle = armAngle;
+			this.shooterSpeed = shooterSpeed;
+		}
+
+        public double getArmAngle() {
+            return armAngle;
+        }
+
+        public double getShooterSpeed() {
+            return shooterSpeed;
+        }
+	}
+	
+	/**
+	* This command performs a shot from a known position
+	* 
+	* @param shooter      the shooter to use
+	* @param arm          the arm to use
+	* @param storageIndex the indexer to use
+	* @param shot         the preset shot to take
+	*/
+	public ShootKnownPos(Shooter shooter, Arm arm, StorageIndex storageIndex, ShotPosition shot) {
+		addRequirements(shooter, arm, storageIndex);
+
+		addCommands(
+			new ParallelCommandGroup(
+				new ArmToPos(arm, shot.getArmAngle()),
+				new SetShooterSpeed(shooter, shot.getShooterSpeed())),
+			new IndexerFeed(storageIndex),
+			new ParallelCommandGroup(
+				new ArmToPos(arm, ArmConstants.stowedSetpoint),
+				new PrepareShooter(shooter, 0)));
```

Comment:
The last parallel command group should have instant commands so we can start another command while the arm and shooter are going back to their setpoints (we don't want to wait for the shooter to slow down all the way back to 0 before another command can use the arm and indexer).